Source/DafnyCore/Backends/SinglePassCodeGenerator/SinglePassCodeGenerator.Statement.cs

```diff
@@ -11,8 +18,6 @@
 
 namespace Microsoft.Dafny.Compilers {
   public abstract partial class SinglePassCodeGenerator {
-
-
     protected void TrStmt(Statement stmt, ConcreteSyntaxTree wr, ConcreteSyntaxTree wStmts = null) {
```

Comment:
Applied a resharper refactoring to turn the if's into a switch statement. No actual changes to this method except for NestedMatchStmt, which now calls into a method.